{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/tests/utils.py

```diff
@@ -0,0 +1,77 @@
+from typing import Tuple
+
+import MDAnalysis as mda
+from MDAnalysis.coordinates.memory import MemoryReader
+import numpy as np
+
+
+def make_Universe(
```

Comment:
How much of this is MDA's own make_Universe?

Even if we switch to LGPL, this would count as "copying" and incur the cost of copyleft.

Reply:
:( most of this is from the original repo. Before I put effort into writing a different thing, what license are we thinking for the cookiecutter? It's MIT currently -- are we happy to keep it that way?

Reply:
1. That file has only ever been touched by four folks: @richardjgowers @orbeckst @mnmelo and myself. If we can get everyone to ok `make_Universe` being released under an MIT license, then we don't need to re-invent the wheel here. 
2. I'm happy to have the cookiecutter be MIT, but until MDA relicenses I think it will auto-GPL by copyleft.

Reply:
As discussed, because we have to be GPL right now, this is not a problem we have to deal with until we have to go MIT (post MDA license change). As such, we can go ahead with what is here and just make sure #20 is closed before we change licenses.